src/styles/blocks/header.scss

```diff
@@ -0,0 +1,158 @@
+@import "../utils/mixins";
+@import "../utils/vars";
+
+.header {
+  width: 100%;
+  box-sizing: border-box;
+  grid-template-rows: 1fr 1fr;
+  align-items: start;
+
```

Comment:
white space

Reply:
I deleted new lines between atributes in css classes.